src/components/twilio-voice-basic-call-control/routes.js

```diff
@@ -7,13 +7,16 @@ const { accountSid, apiKeySid, apiKeySecret } = config;
 const client = Twilio(apiKeySid, apiKeySecret, { accountSid });
 
 router.post('/conferences/:conferenceSid/participants/:callSid', async (req, res) => {
-  const { hold } = req.body;
+  const { hold, muted } = req.body;
   const { callSid, conferenceSid } = req.params;
 
   await client
     .conferences(conferenceSid)
     .participants(callSid)
-    .update({ hold });
+    .update({
+      hold: hold || false,
+      muted: muted || false
```

Comment:
It seems you're setting a default to false? This introduces a bug where if the call is muted, then you put it on hold without passing the muted property again, the call will unmute. Instead of default values, we should not set it if it's undefined.

Reply:
I misunderstood this. I believe just forwarding the `req.body` should work since we do not have to alter any arguments.